foliolabs/foliokit
# Feature/56 dark

This PR closes #56 

## Testing instructions
1. check out this issue/PR locally
2. Be sure to have pulled the latest master for `build-scripts` too:
https://github.com/joomlatools/build-scripts
3. Create a new wordpress site via folioshell and symlink folio-docman 
4. Open up your settings > general options (cmd + space "settings")
5. Switch from dark mode to light mode 

Have a good play about, does any of the html dom/ ui seem off? Anything that doesn't look good under dark mode, or conversly under light mode?
